openshift/templates/cakephp.json

```diff
@@ -230,43 +230,42 @@
   "parameters": [
     {
       "name": "MEMORY_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit",
       "description": "Maximum amount of memory the container can use",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
+      "displayName": "Source Repository URL",
       "description": "The URL of the repository with your application source code",
       "value": "https://github.com/openshift/cakephp-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
-      "displayName": "Source repository reference",
+      "displayName": "Source Repository Reference",
       "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
     },
     {
       "name": "CONTEXT_DIR",
-      "displayName": "Context directory",
+      "displayName": "Context Directory",
       "description": "Set this to the relative path to your project if it is not in the root of your repository"
     },
     {
       "name": "APPLICATION_DOMAIN",
-      "displayName": "Application domain",
+      "displayName": "Application Domain",
       "description": "The exposed hostname that will route to the CakePHP service, if left blank a value will be defaulted.",
       "value": ""
     },
     {
       "name": "GITHUB_WEBHOOK_SECRET",
-      "displayName": "GitHub webhook secret",
+      "displayName": "GitHub Webhook Secret",
       "description": "A secret string used to configure the GitHub webhook",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{40}"
     },
     {
       "name": "DATABASE_SERVICE_NAME",
-      "displayName": "Database service name",
-      "description": "Database service name"
+      "displayName": "Database service name"
```

Comment:
some of the display names are initial caps and others are first word caps only.  is that intentional?